docs/pages/management/dynamic-resources.mdx

```diff
@@ -59,6 +59,10 @@ to manipulate cluster resources stored on the Auth Service backend. The design
 of Teleport's configuration interface makes it well suited for
 infrastructure-as-code and GitOps approaches.
 
+You can get started with `tctl`, the Terraform Provider or the Kubernetes
```

Comment:
I think we should also add a link to the "Manage Access Lists" guide here. A two-item bulleted list would be good for visibility.